src/views/catalog/CreateFromInstanceTypes/components/CreateVMFooter/components/YamlAndCLIViewerModal/YamlAndCLIViewerModal.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FC, KeyboardEvent, MouseEvent as ME, Suspense, useState } from 'react';
```

Comment:
MouseEvent as ME ?

Reply:
since we have [MouseEvent](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent) from JS
and MouseEvent from React, I assigned an alias to avoid React.MouseEvent on line 29.

Reply:
as we talked u can remove some of them, please change in another PR